Koji editor updates (#124)
madewithkoji/docs: src/asciidoc/docs/develop/editor.adoc

> == Previews

Reviewer: Let's add a link to the "Testing" page in this section. Eventually, we'll want to rework this page entirely to be more about "developing in the Koji editor", but that's a project for another day!

You can select the type of preview to display in the preview pane.
The following types are available.

=== Default

Selecting *Default* shows a running preview of your template.

=== Remix

Selecting *Remix* shows what remixers see when customizing values.

=== More

Selecting *More* displays a menu of additional previews.

image::more.png[More menu]

* *About* - Displays an About page, if you've enabled the associated entitlement.

* *Admin* - Displays an Admin page, if you've enabled the associated entitlement.

* *Receipt (seller)* - Displays a preview of the transaction receipt for the seller, if you've implemented in-app purchases and enabled the dynamic receipts entitlement.

* *Receipt (buyer)* - Displays a preview of the transaction receipt for the buyer, if you've implemented in-app purchases and enabled the dynamic receipts entitlement.

* *Sticker* - Displays how your template will look if it's imbedded in another template.

* *Thumbnail* - Displays how your template will look if it's shared on social media.

=== Remote previews

> You can preview your template remotely using the following buttons.

Reviewer: Instead of "remotely", let's describe what we mean by remote previews a little more. These "remote" options are for viewing a preview outside of the editor.

> * The QR code button image:QRcode.png[QR code button] displays a QR code for the template.

Reviewer: Let's add these button screenshots to the "Testing" page, also.

== Previews

You can select the type of preview to display in the preview pane.

> See <<testing-templates.adoc#_using_previews,using previews>>.

Reviewer: ```suggestion
To learn more about using previews, see <<testing-templates#_testing_in_the_koji_editor,Testing in the Koji editor>>.
```
A few problems this suggestion is aiming to address:
* Adding more context so the reader knows what to expect before following the link.
* Fixing Asciidoc markup error (we don't use the `.adoc` in cross-references.

The following types are available.

=== Default

Selecting *Default* shows a running preview of your template.

=== Remix

Selecting *Remix* shows what remixers see when customizing values.

=== More

Selecting *More* displays a menu of additional previews.

image::more.png[More menu]

* *About* - Displays an About page, if you've enabled the associated entitlement.

* *Admin* - Displays an Admin page, if you've enabled the associated entitlement.

* *Receipt (seller)* - Displays a preview of the transaction receipt for the seller, if you've implemented in-app purchases and enabled the dynamic receipts entitlement.

* *Receipt (buyer)* - Displays a preview of the transaction receipt for the buyer, if you've implemented in-app purchases and enabled the dynamic receipts entitlement.

* *Sticker* - Displays how your template will look if it's imbedded in another template.

* *Thumbnail* - Displays how your template will look if it's shared on social media.

=== Remote previews

> The preview pane provides options for previewing a template on different devices or in different environments.

Reviewer: Great info - just need to combine these sentences to make it clearer. E.g.
```suggestion
The preview pane provides these options for previewing a template outside of the Koji editor.
```
